scripts/v.dissolve/v.dissolve.py

```diff
@@ -33,36 +33,339 @@
 # %end
 # %option G_OPT_V_OUTPUT
 # %end
+# %option G_OPT_DB_COLUMN
+# % key: aggregate_column
+# % description: Name of attribute columns to get aggregate statistics for
+# % multiple: yes
+# %end
+# %option
+# % key: aggregate_method
+# % label: Aggregate statistics method
+# % description: Default is all available basic statistics
+# % multiple: yes
+# %end
+# %option G_OPT_DB_COLUMN
+# % key: stats_column
+# % description: New attribute column name for aggregate statistics results
+# % description: Defaults to aggregate column name and statistics name
+# % multiple: yes
+# %end
+# %option
+# % key: aggregate_backend
+# % label: Aggregate statistics method
+# % description: Default is all available basic statistics
+# % multiple: no
+# % required: no
+# % options: univar,sql
```

Comment:
When SQL is a superset of univar, what's the point of supporting univar? GRASS requires SQLite anyway. Users may mix univar methods and SQL functions.